Python/ExcelOutputClass.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+"""
+Spyder Editor
+
+This is a temporary script file.
+"""
+
+import os
+import xlsxwriter
+import openpyxl
+
+os.chdir(r'\Users\Ben\Documents\GitHub\Ben-Projects')
+
+def make_excel_file(title):
```

Comment:
This is dope. I get what you're saying. I am not great at creating array's yet but that's what i'll play with today. I understand your function code, my challenge is getting the data in the proper order / format to feed it into the function.